.github/workflows/docs.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - run: pip install pdoc
       # ADJUST THIS: build your documentation into docs/.
       # We use a custom build script for pdoc itself, ideally you just run `pdoc -o docs/ ...` here.
-      - run: pdoc fsrs --logo "https://joshdavham.com/assets/images/osr_logo.svg" --favicon "https://joshdavham.com/assets/images/osr_logo.svg" -o docs/
+      - run: pdoc fsrs --logo "https://github.com/open-spaced-repetition/py-fsrs/raw/main/osr_logo.png" --favicon "https://github.com/open-spaced-repetition/py-fsrs/raw/main/osr_logo.png" -o docs/
```

Comment:
I’d recommend using a link like this instead:
https://raw.githubusercontent.com/open-spaced-repetition/py-fsrs/refs/heads/{{branch}}/{{file_path}}
![image](https://github.com/user-attachments/assets/327e2ff8-d01d-4a92-ba4c-e8bf7af2302f)


> ref:
> https://github.com/orgs/community/discussions/44370